src/index.ts

```diff
@@ -12,6 +12,14 @@ export type {Metric, MetricSubscriber} from './visuallyCompleteCalculator';
 
 let calculator: VisuallyCompleteCalculator;
 
+const whenActivated = (callback: () => void) => {
+  if (document.prerendering) {
+    window.addEventListener('prerenderingchange', () => callback(), true);
```

Comment:
Nit: You don't need to construct a lambda here.

```suggestion
    window.addEventListener('prerenderingchange', callback, true);
```